<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorView</class>
 <widget class="QMainWindow" name="EditorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>590</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Kimochi Pixlr</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QGraphicsView#drawingCanvas{background-color:white;}QGraphicsView#previewCanvas{background-color:white;}
QMainWindow#EditorView{background-color:rgb(230, 230, 250);}
QWidget#centralwidget{background-color:rgb(0, 128, 128);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="XYcoordinateLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bangla MN</family>
     </font>
    </property>
    <property name="text">
     <string>X:     Y:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="drawingCanvas">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="previewCanvas">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>230</width>
      <height>230</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="viewFrameLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bangla MN</family>
     </font>
    </property>
    <property name="text">
     <string>View Frames</string>
    </property>
   </widget>
   <widget class="QLabel" name="drawingLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>401</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="previewLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>230</width>
      <height>230</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bangla MN</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>KIMOCHI PIXLR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="addFrameButton">
       <property name="font">
        <font>
         <family>Bangla MN</family>
        </font>
       </property>
       <property name="text">
        <string>ADD FRAME +</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="frameSelectionsBox">
       <property name="font">
        <font>
         <family>Bangla MN</family>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteFrameButton">
       <property name="font">
        <font>
         <family>Bangla MN</family>
        </font>
       </property>
       <property name="text">
        <string>DELETE FRAME -</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>191</width>
      <height>130</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="fpsLabel">
         <property name="font">
          <font>
           <family>Bangla MN</family>
          </font>
         </property>
         <property name="text">
          <string>Fps: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="fpsSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="PixelsizeLabel">
         <property name="font">
          <font>
           <family>Bangla MN</family>
          </font>
         </property>
         <property name="text">
          <string>Pixel Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="pixelSizeSlider">
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>drawingCanvas</zorder>
   <zorder>drawingLabel</zorder>
   <zorder>XYcoordinateLabel</zorder>
   <zorder>previewCanvas</zorder>
   <zorder>viewFrameLabel</zorder>
   <zorder>previewLabel</zorder>
   <zorder>label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionPen"/>
    <addaction name="actionEraser"/>
    <addaction name="actionMirror"/>
    <addaction name="actionTransform"/>
    <addaction name="actionColor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionBasic_Features"/>
    <addaction name="actionExtra_Features"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPen"/>
   <addaction name="actionEraser"/>
   <addaction name="actionMirror"/>
   <addaction name="actionTransform"/>
   <addaction name="actionColor"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Meta+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Meta+S</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/pen.png</normaloff>:/images/pen.png</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/erase.png</normaloff>:/images/erase.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionTransform">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/transform.png</normaloff>:/images/transform.png</iconset>
   </property>
   <property name="text">
    <string>Transform</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionMirror">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/mirror.png</normaloff>:/images/mirror.png</iconset>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/paintbrush.png</normaloff>:/images/paintbrush.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionBasic_Features">
   <property name="text">
    <string>Drawing Features</string>
   </property>
  </action>
  <action name="actionExtra_Features">
   <property name="text">
    <string>File/Frame Management</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
